Tools/setup/requirements.txt

```diff
@@ -0,0 +1,8 @@
+argparse>=1.2
```

Comment:
Need to install 
```
setuptools>=39.2.0
wheel>=0.31.1
```
(or `python-setuptools` via apt for those tools). Otherwise script gives errors.

Reply:
Building toml etc creates wheels by default, so if you don't have wheel installed you get an error for that.

I can't remember the setuptools errors, but was coming in same place.